src/App.tsx

```diff
@@ -1,25 +1,33 @@
-import React from "react";
-import Header from "./components/Header/Header";
-import dfu from './dfu-util-js/dfu';
+import React, { useState } from "react";
+import dfu from "./dfu-util-js/dfu";
+import "./App.scss";
+import { Header } from "./components/Header/Header";
```

Comment:
As I mentioned in my other comment, once the `Header.tsx` file uses the `export default` syntax, you can then remove the brackets in `{ Header }` and just import it like this:
```
import Header from "./components/Header/Header";
```